quixstreams/dataframe/dataframe.py

```diff
@@ -126,19 +131,24 @@ class StreamingDataFrame:
 
     def __init__(
         self,
-        topic: Topic,
+        *topics: Topic,
         topic_manager: TopicManager,
-        registry: DataframeRegistry,
+        registry: DataFrameRegistry,
         processing_context: ProcessingContext,
         stream: Optional[Stream] = None,
     ):
+        if not topics:
+            raise ValueError("At least one Topic must be passed")
+
         self._stream: Stream = stream or Stream()
-        self._topic = topic
+        self._topics = set(topics)
```

Comment:
If you don't apply the other suggestion about this line, alternatively, let's use a frozenset to indicate that we don't allow mutating topics after sdf instantiation.
```suggestion
        self._topics = frozenset(topics)
```